README.md

```diff
@@ -0,0 +1,31 @@
+To run the model, please download the following pre-trained models and put them in /pre-trained_models
+
+Wav2vec_large.pt:
+https://drive.google.com/file/d/1VgvNHpY9N-guw0z96uP1r2Cw7iuiM2oV/view?usp=drive_link
+
+
+Affectnet_4views.pth:
+https://drive.google.com/file/d/1Pe4bQ01Umh8f6W_diU16KFccg4DopF3P/view?usp=drive_link
+
+
+RobBERT-base.pt:
+https://drive.google.com/file/d/1pDLLxSjNiYslWkwESDFkcGT-S_f64rAK/view?usp=drive_link
+
+Please also download the checkpoint for SSEFT and put it in /checkpoint:
+https://drive.google.com/file/d/1AL3RktxPNv-ZU3XV9yoR24mu1W1J4SS9/view?usp=drive_link
+
+
+You can put any video/audio file pair in /Structured_input_data, but to test you can use these files:
+audio:
+https://drive.google.com/file/d/1azocILZpCmUL1j2nzfRJwvbd-jgglTFX/view?usp=drive_link
+video:
+https://drive.google.com/file/d/1-Sln1Uywee1n7A65E-b0IdnXWzSre8iI/view?usp=drive_link
+
+
+Build in docker: 
+docker build dane-emotion-recognition-worker
+
+
+Run in container with (change path for sources! and build id):
+docker run     --mount type=bind,source="/home/kdeboer/dane-emotion-recognition-worker/config",target=/root/.DANE     --mount type=bind,source="/home/kdeboer/dane-emotion-recognition-worker/data",target=/data     --mount type=bind,source="/home/kdeboer/dane-emotion-recognition-worker/Structured_input_data",target=/Structured_input_data     --mount type=bind,source="/home/kdeboer/dane-emotion-recognition-worker/pre-trained_models",target=/pre-trained_models     --mount type=bind,source="/home/kdeboer/dane-emotion-recognition-worker/checkpoint",target=/checkpoint     --rm         cf222214c6d3 --run-test-file
```

Comment:
Use the image tag (dane-emotion-recognition-worker) instead of its id (cf222214c6d3)